VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift

```diff
@@ -150,6 +146,8 @@ struct VaultDetailView: View {
         viewModel.setOrder()
         viewModel.updateBalance()
         viewModel.getGroupAsync(tokenSelectionViewModel)
+        viewModel.setDefaultCoins(for: vault)
+        appState.currentVault = vault
```

Comment:
`viewModel.setDefaultCoins` if put in here , it will be called multiple time , which will cause it to crash , you need to put it to onAppear, and only call it once